mysite/settings.py

```diff
@@ -25,7 +26,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['0', 'localhost']
```

Comment:
I'd recommend `ALLOWED_HOSTS = ['*']` for accepting connections from any interfaces.

Reply:
Updated.